skymap_scanner/utils/extract_json_message.py

```diff
@@ -37,11 +37,13 @@ def extract_GCD_diff_base_filename(frame_packet):
 
             if GCD_diff_base_filename is None:
                 GCD_diff_base_filename = frame[key].base_filename
+                LOGGER.debug(f"GCD diff base_filename loaded from {key} in {frame.Stop} frame.")
             elif frame[key].base_filename != GCD_diff_base_filename:
                 raise RuntimeError("inconsistent frame diff base GCD file names. expected {0}, got {1}".format(GCD_diff_base_filename, frame[key].base_filename))
 
     if GCD_diff_base_filename == "current-gcd":
-        LOGGER.warning(" **** WARNING: baseline GCD file is \"current-gcd\". replacing with \"2016_01_08_Run127381.i3\".")
+        # It is unclear which legacy case is covered by this condition.
+        LOGGER.warning("Baseline GCD file is \"current-gcd\". Replacing with \"2016_01_08_Run127381.i3\".")
         GCD_diff_base_filename = "2016_01_08_Run127381.i3"
```

Comment:
this is concerning, but perhaps we can't avoid it

Reply:
It seems that this applies only to very old alerts. It is very realtime-specific so I expect this will be decoupled from Skymap Scanner in the future.